variables.tf

```diff
@@ -84,19 +84,24 @@ variable "enable_http_endpoint" {
 
 variable "engine" {
   type        = string
-  default     = "aurora"
+  default     = "aurora-mysql"
   description = "The engine type of the Aurora cluster"
 }
 
 variable "engine_mode" {
   type        = string
   default     = "serverless"
   description = "The engine mode of the Aurora cluster"
+
+  validation {
+    condition     = contains(["provisioned", "serverless", "parallelquery", "global", "multimaster", "serverlessv2"], var.engine_mode)
+    error_message = "Allowed values for engine_mode are \"provisioned\", \"serverless\", \"parallelquery\", \"global\", \"multimaster\" or \"serverlessv2\"."
+  }
 }
 
 variable "engine_version" {
   type        = string
-  default     = "5.6.10a"
+  default     = "5.7.mysql_aurora.2.08.3"
```

Comment:
rather than updating the engine_version all the time, does it not make more sense to delete the default? Making this a mandatory variable?

Reply:
Mainly because it's a drama to find the correct values. This works "out of the box"